edlib/src/edlib.tpp

```diff
@@ -228,11 +213,12 @@ EdlibAlignResult edlibAlign(const Element* const queryOriginal, const int queryL
     /*--------------------- INITIALIZATION ------------------*/
     int maxNumBlocks = ceilDiv(queryLength, WORD_SIZE); // bmax in Myers
     int W = maxNumBlocks * WORD_SIZE - queryLength; // number of redundant cells in last level blocks
+    std::vector<EdlibEqualityPair<AlphabetIdx>> additionalEqualities =
+            transformEqualityPairs(config.additionalEqualities,
+                                   config.additionalEqualitiesLength,
+                                   elementToAlphabetIdx);
     EqualityDefinition<AlphabetIdx> equalityDefinition =
-            makeEqualityDefinition<Element, AlphabetIdx>(config,
-                                                  elementToAlphabetIdx,
-                                                  config.additionalEqualities,
-                                                  config.additionalEqualitiesLength);
+        EqualityDefinition<AlphabetIdx>(additionalEqualities, elementToAlphabetIdx.size());
```

Comment:
I believe you can just do inline: `EqualityDefinition<AlphabetIdx>(additionalEqualities, elementToAlphabetIdx.size());`, no need for assignment, you are just doing redundant copying possibly.